### ISSUE TITLE
Add visual cues for polygon alignment

### ISSUE THREAD

**Issue by SetField** [synthetiv/euclidigons#2]
Visual cue (small “tics” at top and bottom of screen) when the center of the polygon you're moving is aligned with the center or left/right boundary of another polygon.

**Comment by synthetiv** [comment]
interesting. I haven't felt a strong need for this but maybe I will when I try it. Is the idea to make it clearer when polygons do or don't overlap, or to make it easier to precisely align them so only their points will touch? or neither/both?

**Comment by SetField** [comment]
Yes to both! I feel like a visual clue to help me know when polygons are centered and when they just touch would help the user have a reference point for setting the amount of collisions. Am I making sense?
